scripts/types.ts

```diff
@@ -48,12 +48,17 @@ export interface TokenPriceData {
   priceFetchedAt: number
 }
 
+export interface KpiResult {
+  kpi: number
+  segmentedKpi?: { [key: string]: number }
+}
```

Comment:
```suggestion
/**
 * Represents the result of a KPI computation.
 *
 * @template T - The allowed string keys for the breakdown.
 * - `value`: overall KPI value.
 * - `breakdown`: an optional map from typed segment keys to their KPI values.
 */
export interface KpiResult<T extends string = string> {
  value: number
  breakdown?: Record<T, number>
}
```

This gives us:
- Type safety for known segments per KPI function
- Cleaner dev experience via autocomplete and key validation
- Better documentation
- Flexibility, still works with untyped keys when the generic is omitted

Then we could use it like this:

```ts
type ScoutGameBreakdown = 'base' | 'celo' | 'polygon'

const result: KpiResult<ScoutGameBreakdown> = {
  value: 87,
  breakdown: {
    base: 90,
    celo: 85,
    polygon: 42,
  },
}
```

Reply:
oh, good suggestion @jeanregisser :bow: